admin/packaging.py

```diff
@@ -754,6 +764,14 @@ class PACKAGE_NODE(PACKAGE):
     )
 
 
+class PACKAGE_DOCKER_PLUGIN(PACKAGE):
+    DESCRIPTION = ValueConstant(
+        'Volume plugin for Docker\n'
+        + fill('This meta-package contains links to the Flocker Docker plugin',
+               79)
```

Comment:
:question: Maybe say "Flocker volumes plugin for Docker"

Here's how it looks to the installer:

```
root@acceptance-test-richardw-0:~# aptitude search flocker 
p   clusterhq-flocker-cli                                              - Docker orchestration and volume management tool                             
p   clusterhq-flocker-docker-plugin                                    - Volume plugin for Docker                                                    
i   clusterhq-flocker-node                                             - Docker orchestration and volume management tool                             
i A clusterhq-python-flocker                                           - Docker orchestration and volume management tool                             
root@acceptance-test-richardw-0:~# aptitude show clusterhq-flocker-docker-plugin
Package: clusterhq-flocker-docker-plugin 
State: not installed
Version: 1.2.0-0.dev.1.152.g94a1d7d
Priority: extra
Section: admin
Maintainer: ClusterHQ <contact@clusterhq.com>
Architecture: all
Uncompressed Size: 0 
Depends: clusterhq-python-flocker (= 1.2.0-0.dev.1.152.g94a1d7d)
Description: Volume plugin for Docker
 This meta-package contains links to the Flocker Docker plugin
Homepage: https://clusterhq.com

```